### PR TITLE
HOTT-2299: Fixed country dropdown on commodities page

### PR DESCRIPTION
**Pull request by maverickvi**
### Jira link

[HOTT-<TODO>](https://transformuk.atlassian.net/browse/HOTT-2299)

### What?

I have added/removed/altered:

- [ ] Fixed country dropdown on commodities page

### Why?

I am doing this because:

- it was rendering incorrectly and pushing page content downwards

### Have you? (optional)

- [ ] Validated mobile responsive behaviour of view changes

Screenshot 2023-02-06 at 15 19 46

### CODE REVIEW

**Review by willfish — APPROVED** [review]

**Review by jebw — CHANGES_REQUESTED** [review]
I think this is wrong - this is replacing the inline css to behave like overlay _but_ the issue is we want the overlay behaviour, so we should change the javascript to use the overlay config option

https://alphagov.github.io/accessible-autocomplete/#other-options

File: app/webpacker/src/javascripts/commodities.js [hunk 1]
```diff
@@ -752,6 +752,7 @@
                 showAllValues: true,
                 confirmOnBlur: true,
                 alwaysDisplayArrow: true,
+                displayMenu: 'overlay',
```
Comment (willfish) [inline app/webpacker/src/javascripts/commodities.js:755]: Do we have any javascript tests for this? Is it appropriate to have coverage here?

**Review by willfish — APPROVED** [review]
LGTM assuming @jebw is happy

File: app/webpacker/src/javascripts/commodities.js (hunk 1 above)
Comment (maverickvi) [reply app/webpacker/src/javascripts/commodities.js:755]: This is provided by the add-on so we can just assume they have throughly tested this feature

Comment (willfish) [reply app/webpacker/src/javascripts/commodities.js:755]: Yeah, that's fair enough - this is essentially configuration

**Review by jebw — APPROVED** [review]
LGTM :raised_hands:

**Pull request merged**